### ISSUE TITLE
not perfect yet lmao

### ISSUE THREAD

**Issue by Kevoting** [dank074/music-player#1]
the habstep breaks this

1:672,12;673,2;0,2;672,12;673,2;0,2;672,12;0,12;672,4;673,2;0,2;672,12;672,3;0,1;672,12;673,2;0,10;672,12;673,2;0,2;672,12;673,2:2:671,44;0,4;671,48;0,22;18,2;0,7;81,3:3:0,8;675,38;0,18;675,22;675,1;0,5;675,4;0,23;20,8;21,1;20,6;21,1:4:0,12;670,4;0,15;20,4;26,1;20,3;21,1;20,4;26,1;0,2;20,1;21,3;20,1;21,1;20,1;21,1;0,1;20,6;21,1;73,12;20,1;73,2;73,1;0,1;670,4;0,18;670,4;0,4;670,4;0,4;670,4;0,4;670,4:5:0,16;676,12;676,3;0,1;676,12;676,3;0,5;676,8;685,4;676,20;676,2;0,2;676,14;0,2;676,12;676,3;0,1;676,4;676,3;0,1;676,4;676,3:6:0,32;670,28;0,4;670,8;0,4;670,4;0,8;670,8;0,1;43,1;0,6;669,40:7:0,31;15,1;0,8;681,1;0,5;680,2;678,2;679,2;678,2;679,2;678,2;679,2;678,2;679,2;684,4;681,1;0,1;683,2;684,4;681,1;0,1;683,2;684,2;682,2;684,6;683,2;684,2;682,2;684,8;0,8;36,2;0,10;36,2;0,2;36,2;0,2;36,2:8:0,14;15,1;17,1;0,1;56,2;0,11;18,2;0,12;686,4;0,27;15,1;0,3;18,2;0,19;677,4;0,12;81,3;0,1;674,16;674,3:meta,1;c,1

**Comment by s074** [comment]
Can you elaborate? I have no idea what that song is supposed to sound like

**Comment by Kevoting** [comment]
> Can you elaborate? I have no idea what that song is supposed to sound like

put the song data in flash and play, now play here you will notice that some sounds don't start, the song is Official and is called 'The HabStep'

**Comment by Kevoting** [comment]
here is a reference
https://www.youtube.com/watch?v=owZ3YkyigEM

**Comment by s074** [comment]
You are right. It could be that too many sounds are playing at once and it reaches the browser's limit on concurrent sounds playing. I think the only solution to this would be to use the audio context to merge each individual sound into a single one and then play that 🤔

**Comment by DavydeVries** [comment]
Maybe late, the thing is that your example is pointing to:

```
https://flash.nitrodev.co/dcr/hof_furni/mp3/sound_machine_sample_%sample%.mp3
```
That source is missing the "not released" samples, ranging from `649-731`.

When pointing to my own source (with those not released samples), your player is just working fine, even the extended versions.
